pom.xml

```diff
@@ -319,6 +319,14 @@
                     </execution>
                 </executions>
             </plugin>
+            <plugin>
```

Comment:
Do you think that this is needed? Any idea how was it working before?

Reply:
It didn't, this was added to support java 7 features such as diamond operator. I removed it and added explicit arguments..